pkg/interface/groups/src/js/components/lib/add-contact.js

```diff
@@ -63,7 +63,7 @@ export class AddScreen extends Component {
     return (
       <div className="h-100 w-100 flex flex-column overflow-y-scroll">
         <div className="w-100 dn-m dn-l dn-xl inter pt1 pb6 pl3 pt3 f8">
-          <Link to={"/~contacts" + props.path}>{"⟵ All Contacts"}</Link>
+          <Link to={"/~groups" + props.path}>{"⟵ All Contacts"}</Link>
```

Comment:
All Groups :)

Reply:
All Groups is the first sidebar. This mobile nav takes you back to the second one, of the contacts in the group.

Reply:
kk